app/validators/email_validator.rb

```diff
@@ -0,0 +1,7 @@
+class EmailValidator < ActiveModel::EachValidator
+  def validate_dach(record, attribute, value)
+    unless value =~ /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\z/i
+      record.errors[attribute] << (options[:message] || "メールアドレスの形式が正しくありません。")
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols.